harmony/http.py

```diff
@@ -32,6 +34,10 @@
 # https://2.python-requests.org/en/master/user/quickstart/#timeouts
 TIMEOUT = 60
 
+# Error codes for which we should retry failed requests
+RETRY_ERROR_CODES = (408, 502, 503, 504)
```

Comment:
Does this affect just downloading files or also the requests calling back to harmony?

Reply:
It's used by the retry adapter which is used in the download function and in validating the EDL token.